docs/getting-started.md

```diff
@@ -22,23 +13,40 @@ $ brew install semgrep
 # For Ubuntu / Windows via WSL / Linux / macOS
 $ python3 -m pip install semgrep
 
-# To try Semgrep without installation run via Docker
-$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep --help
+# For Docker users
+$ docker pull returntocorp/semgrep
 ```
 
-Confirm installation and run both a simple “grep-like” rule and a full ruleset:
+Next, navigate to a local code repository and run Semgrep using a set of rules from the public Semgrep Registry. Analysis executes fully on your computer: **code is never uploaded anywhere**.
 
 ```sh
```

Comment:
TODO - Explain how to invoke this via Docker (since we just have Docker pull up top now)